src/server/middlewares/chats/postChat.js

```diff
@@ -0,0 +1,14 @@
+const axios = require('axios');
+
+const postChat = text => {
+  const res = axios.post(
+    'https://hooks.slack.com/services/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',
+    {
+      text: text,
+    }
+  );
+
+  return res.data;
+};
```

Comment:
What if this request fails? shouldn't it be wrapped in some try/catch, or gracefully return errors.

Reply:
True, Will add that 🙂

Reply:
This has been handled now